docs/NFTs.md

```diff
@@ -0,0 +1,69 @@
+# Non-Fungible Tokens (NFTs)
+
+## Features
+
+- **NFT Ownership**: Indicates which address currently owns a particular NFT.
+- **NFT Metadata**: Displays metadata associated with the NFT, such as name,
+  description, image defined in the token's contract.
+- **Transaction History**: Shows the transaction history of the NFT, allowing
+  you to see past owners and sales prices.
+- **Minting Information**: The time and block of when NFT was minted.
+
+## NFT Metadata support
+
+As of the time this documentation was written, our Explorer is designed to
+support only the
+official [ERC-721](https://github.com/ethereum/ERCs/blob/master/ERCS/erc-721.md)
+metadata standard, as per Ethereum's
+specifications.
+
+At present, our system primarily supports the display of image type assets with
+a preview feature. These image types can
+reside on a variety of hosting platforms, including HTTP, HTTPS, FTP, IPFS, or
```

Comment:
Right. Will remove `http` to avoid mixed content. As for `ftp` I just looked it up, I know there was in-browser support a while ago, but looking at it, it was discontinued in 2021.